### PR TITLE
doc: Update 'Netplan everywhere' for 23.10

### PR DESCRIPTION
**Pull request by slyon**
## Description
Dropping the PPA parts and update the workflow for Ubuntu 23.10, in preparation for an upcoming blog post.

## Checklist

- [x] Runs `make check` successfully.
- [x] Retains 100% code coverage (`make check-coverage`).
- [ ] New/changed keys in YAML format are documented.
- [ ] \(Optional\) Adds example YAML for new feature.
- [ ] \(Optional\) Closes an open bug in Launchpad.

### CODE REVIEW

File: doc/netplan-everywhere.md [hunk 1]
```diff
@@ -1,92 +1,86 @@
...
-to store network configuration. In the Desktop it's convenient to use
-graphical tools for configuration when they are available, so nothing will
-change from the user perspective, only the way the configuration is
-handled under the hood.
+On Ubuntu 23.10 "Mantic Minotaur" and later, NetworkManager uses Netplan's APIs
+to save the configuration created through any of its graphical or programmatic
+interfaces. This leads to having a centralized location to store network
+configuration. In the Desktop it's convenient to use graphical tools for
+configuration when they are available, so nothing will change from the user
+perspective, only the way the configuration is handled under the hood.
```
Comment (rkratky) [inline doc/netplan-everywhere.md:16]:
```suggestion
perspective; only the way the system handles the configuration in the background.
```

File: doc/netplan-everywhere.md [hunk 2]
```diff
@@ -1,92 +1,86 @@
...
 
-For more information on Netplan, check https://netplan.io/
+For more information on Netplan, check [https://netplan.io](https://netplan.io)
```
Comment (rkratky) [inline doc/netplan-everywhere.md:18]:
```suggestion
For more information on Netplan, see [https://netplan.io](https://netplan.io).
```

File: doc/netplan-everywhere.md [hunk 3]
```diff
@@ -1,92 +1,86 @@
...
 
-For more information on NetworkManager, check https://networkmanager.dev/
+For more information on NetworkManager, check [https://networkmanager.dev](https://networkmanager.dev)
```
Comment (rkratky) [inline doc/netplan-everywhere.md:20]:
```suggestion
For more information on NetworkManager, see [https://networkmanager.dev](https://networkmanager.dev).
```

File: doc/netplan-everywhere.md [hunk 4]
```diff
@@ -1,92 +1,86 @@
...
 
 ## How it works
 
 Every time a non-temporary connection is created in NetworkManager, instead
-of persisting the original .nmconnection file, it will create a Netplan YAML
-at `/etc/netplan` called `90-NM-<connection UUID>.yaml`. After creating the
+of persisting the original `.nmconnection` file, it will create a Netplan YAML
```
Comment (rkratky) [inline doc/netplan-everywhere.md:25]:
```suggestion
of persisting the original `.nmconnection` file, it creates a Netplan YAML
```

**Review by rkratky — APPROVED** [review]
@slyon Some lang. suggestions.

**Comment by slyon** [comment]
Thank you, merging!

**Pull request merged**